src/main/resources/marc4bibframe.yml

```diff
@@ -1705,3 +1640,91 @@ bibSharedRules:
             CODE: 15,18
         constants:
           LINK: http://id.loc.gov/vocabulary/countries/%CODE
+  776_mappings:
```

Comment:
Let us rename this to "linking_entries_mappings" or "7XX_mappings"

Reply:
Done